README.md

```diff
@@ -86,6 +86,7 @@ Awesome resources for [Relay](https://github.com/facebook/relay), based on the [
 - [Relay Skeleton](https://github.com/fortruce/relay-skeleton) - React, Relay, GraphQL project skeleton.
 - [Relay Starter Kit](https://github.com/relayjs/relay-starter-kit) - An app that it already set up with a basic setup. Just clone and tweak to suit your needs!
 - [Simple Relay Starter](https://github.com/mhart/simple-relay-starter) - A Browserify version of the [Relay Starter Kit](https://github.com/relayjs/relay-starter-kit).
+- [Relay Fullstack](https://github.com/lvarayut/relay-fullstack) - Relay Starter Kit integrated with Relay, GraphQL, Express, ES6/ES7, JSX, Webpack, Babel, Material Design Lite, and PostCSS.
```

Comment:
This is super nit-picky, but some of this stuff can be assumed. Relay is based on GraphQL, for instance. I'm open to the current version, but how about something along the lines of:

```markdown
The `Relay Starter Kit`, integrated with Relay, Express, Material Design Lite, PostCSS, Babel, and built with Webpack
```